schemas/test/en/test_supplementary_data_multiple_datasets.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
Do we really need this schema?. Adding a runner schema just to test launcher doesn't seem worth it. Just have multiple for the main one, that proves both cases really?

Reply:
Yep thats a good point - we can just change the survey_id in metadata to see the one with multiple datasets come through and have the single schema already there